app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
Why removing this? Because it's not used or because the adapter changed in the box?